### PR TITLE
release 1.31.1

### PR DESCRIPTION
**Pull request by cqc-melf**
# Description

Please summarise the changes.

# Related issues

Please mention any github issues addressed by this PR.

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: pytket/docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+1.31.1 (August 2024)
+--------------------
+
+* Fix issue in pytket-circuit-renderer 0.9.
```
Comment (CalMacCQ) [inline pytket/docs/changelog.rst:7]: Fix issue with css styles in pytket-circuit-renderer 0.9. ?

**Review by CalMacCQ — APPROVED** [review]

File: pytket/docs/changelog.rst (hunk 1 above)
Comment (cqc-melf) [reply pytket/docs/changelog.rst:7]: Not sure if we need that much details, happy to update that if you want to.

Comment (CalMacCQ) [reply pytket/docs/changelog.rst:7]: Up to you, its a minor change. I'm fine with either.

Comment (cqc-melf) [reply pytket/docs/changelog.rst:7]:
```suggestion
* Fix issue with css styles in pytket-circuit-renderer 0.9.

**Review by CalMacCQ — APPROVED** [review]

**Comment by cqc-melf** [comment]
Currently checking if we really need this. Will delay it if possible.

**Pull request merged**